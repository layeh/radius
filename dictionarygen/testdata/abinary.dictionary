ATTRIBUTE ABinaryAttr 123 abinary
