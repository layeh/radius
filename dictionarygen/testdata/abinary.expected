Constants:
	const (
		ABinaryAttr_Type radius.Type = 123
	)
Functions:
		func ABinaryAttr_Add(p *radius.Packet, value []byte) (err error)
		func ABinaryAttr_AddString(p *radius.Packet, value string) (err error)
		func ABinaryAttr_Del(p *radius.Packet)
		func ABinaryAttr_Get(p *radius.Packet) (value []byte)
		func ABinaryAttr_GetString(p *radius.Packet) (value string)
		func ABinaryAttr_GetStrings(p *radius.Packet) (values []string, err error)
		func ABinaryAttr_Gets(p *radius.Packet) (values [][]byte, err error)
		func ABinaryAttr_Lookup(p *radius.Packet) (value []byte, err error)
		func ABinaryAttr_LookupString(p *radius.Packet) (value string, err error)
		func ABinaryAttr_Set(p *radius.Packet, value []byte) (err error)
		func ABinaryAttr_SetString(p *radius.Packet, value string) (err error)
